### PR TITLE
feat: new rules related to CVE-2023-36884

### PR DESCRIPTION
**Pull request by X-Junior**
### Summary of the Pull Request
rules for CVE-2023-36884 based on
https://blogs.blackberry.com/en/2023/07/romcom-targets-ukraine-nato-membership-talks-at-nato-summit

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Review by nasbench — APPROVED** [review]

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/file_event_win_exploit_cve_2023_36884_office_windows_html_rce_file_patterns.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+title: Potential CVE-2023-36884 Exploitation Dropped Files
+id: 8023d3a2-dcdc-44da-8fa9-5c7906e55b38
+status: experimental
+description: Detects specific file pattern being created in the office recent folder. Seen being dropped during potential exploitation of CVE-2023-36884
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/file_event_win_exploit_cve_2023_36884_office_windows_html_rce_file_patterns.yml:4]:
```suggestion
description: Detects a specific file being created in the recent folder of Office. These files have been seen being dropped during potential exploitations of CVE-2023-36884
```

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/proxy_exploit_cve_2023_36884_office_windows_html_rce.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+title: Potential CVE-2023-36884 Exploitation Pattern
+id: 0066d244-c277-4c3e-88ec-9e7b777cc8bc
+status: experimental
+description: Detects unique pattern seen being used by RomCom potentially exploiting CVE-2023-36884
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/proxy_exploit_cve_2023_36884_office_windows_html_rce.yml:4]:
```suggestion
description: Detects a unique pattern seen being used by RomCom potentially exploiting CVE-2023-36884
```

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/proxy_exploit_cve_2023_36884_office_windows_html_rce_extenstion_ip_pattern_traffic.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+title: Potential CVE-2303-36884 URL Request Pattern Traffic
+id: d9365e39-febd-4a4b-8441-3ca91bb9d333
+status: experimental
+description: Detects specific URL pattern containing a specific extension and parameters pointing to an IP address, this pattern was seen being used by RomCOM in potential exploitation of CVE-2023-36884
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/proxy_exploit_cve_2023_36884_office_windows_html_rce_extenstion_ip_pattern_traffic.yml:4]:
```suggestion
description: Detects a specific URL pattern containing a specific extension and parameters pointing to an IP address. This pattern was seen being used by RomCOM potentially exploiting CVE-2023-36884
```

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/file_event_win_exploit_cve_2023_36884_office_windows_html_rce_file_patterns.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+title: Potential CVE-2023-36884 Exploitation Dropped Files
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/file_event_win_exploit_cve_2023_36884_office_windows_html_rce_file_patterns.yml:1]:
```suggestion
title: Potential CVE-2023-36884 Exploitation Dropped File
```

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/proxy_exploit_cve_2023_36884_office_windows_html_rce_url_marker_traffic.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+title: Potential CVE-2023-36884 Exploitation - URL Marker
+id: e59f71ff-c042-4f7a-8a82-8f53beea817e
+status: experimental
+description: Detects unique URL marker seen being used by RomCom potentially exploiting CVE-2023-36884
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/proxy_exploit_cve_2023_36884_office_windows_html_rce_url_marker_traffic.yml:4]:
```suggestion
description: Detects a unique URL marker seen being used by RomCom potentially exploiting CVE-2023-36884
```

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml [hunk 6]
```diff
@@ -0,0 +1,28 @@
+title: Potential CVE-2023-36884 Exploitation - Share Access
+id: 3df95076-9e78-4e63-accb-16699c3b74f8
+status: experimental
+description: Detects access to a specific share pattern seen being used during exploitation of CVE-2023-36884
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml:4]:
```suggestion
description: Detects access to a file share with a naming schema seen being used during exploitation of CVE-2023-36884
```

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    definition: 'The advanced audit policy setting "Object Access > Audit File Share" must be configured for Success/Failure'
+detection:
+    selection_eid:
+        EventID: 5140
+    selection_share_name:
+        ShareName|contains: '\\\\MSHTML_C7\\\\'
+        ShareName|re: '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}'
+    selection_share_path:
+        ShareLocalPath|contains: '\\\\MSHTML_C7\\\\'
+        ShareLocalPath|re: '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}'
```
Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml:24]: Wanna make sure that you want to match a literal `\\MSHTML_C7\\`. My guess is, that the 5140 event does some escaping itself?

Comment (phantinuss) [inline rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml:24]:
```suggestion
        ShareLocalPath|re: '\\\\[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}'
```
not sure about the format of the 5140 event, but this should be more specific, if it works.

File: tests/cti
[the remark this answers is not in the document]
Comment (phantinuss) [reply tests/cti:1]: That's probably also the reason that (by chance) the sigma test for ATTACK mapping is failing. It's using outdated data.

File: rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml (hunk 7 above)
Comment (nasbench) [reply rules-emerging-threats/2023/Exploits/CVE-2023-36884/win_security_exploit_cve_2023_36884_office_windows_html_rce_share_access_pattern.yml]: I just removed all escaping to make it catch all possible cases. I don't have enough information about how people collect it and the EID itself doesn't do the escaping example ` <Data Name="ShareName">\\*\IPC$</Data>`

File: rules/windows/network_connection/net_connection_win_office_susp_ports.yml [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+            - '\wordview.exe'
+    filter_main_ports:
+        DestinationPort:
+            - 139
+            - 443
+            - 445
+            - 80
+    condition: selection and not 1 of filter_main_*
+falsepositives:
+    - Other ports can be used, apply additional filter accordingly
```
Comment (phantinuss) [inline rules/windows/network_connection/net_connection_win_office_susp_ports.yml:32]:
```suggestion
    - Other ports can be used, apply additional filters accordingly
```

**Review by phantinuss — APPROVED** [review]

**Pull request merged**